api/v1/routes.go

```diff
@@ -138,6 +139,15 @@ func postLayer(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx
 }
 
 func getLayer(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *context.RouteContext) (string, int) {
+
+	log.Info(updater.DbFinishUpdate)
+
+	if updater.DbFinishUpdate {
+		writeResponse(w, r, http.StatusNotFound, LayerEnvelope{Error: & Error{"DB is being updated so no query to it can be done"}})
```

Comment:
@Quentin-M I added the changes that you requested on the error code and message.